src/text/text_value.rs

```diff
@@ -94,7 +94,7 @@ impl TextValue {
     }
 }
 
-/// Converts a given type into a `Vec<OwnedSymbolToken>`.
+/// Converts a given type into a `Vec<RawSymbolToken>`.
```

Comment:
🗺️ This was a dangling documentation reference to the long ago removed `OwnedSymbolToken` type.